### DISCUSSION TITLE
Indexing directories containing subdirectories

### DISCUSSION THREAD

**Discussion by tominal** [joshembling/laragenie#16] [Q&A] [answered]
Apologies if I missed this somewhere in the docs. When indexing, it looks like it is only indexing the files located in the directory I provide and not all the subdirectories and files within it.

I have tried the following in my config:

```php
 'indexes' => [
        ...
        'directories' => [ 'resources/*', 'resources/js/*' ],
        ...
]
```

Any ideas?

**Comment by joshembling** [comment, marked as answer]
Hi @tominal 

Right now, that functionality is not possible to search through every sub directory 

Snippet from the docs on current functionality:

- Inputting a full directory, e.g. App
    - If you pass in a directory, Laragenie can only index files within this directory, and not its subdirectories.
    - To index subdirectories you must explicitly pass the path e.g. `app/Models` to index all of your models
